pcap/org.eclipse.tracecompass.tmf.pcap.core/src/org/eclipse/tracecompass/internal/tmf/pcap/core/util/ProtocolConversion.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2014 Ericsson
+ * 
```

Comment:
Do not remove copyright headers from these files, you can put 2014, 2018 Ericsson, your organization